src/engraving/types/symnames.cpp

```diff
@@ -3083,7 +3083,48 @@ constexpr const std::array<AsciiStringView, size_t(SymId::lastSym) + 1> SymNames
     "noteLongaDown",
     "noteLongaSquareUp",
     "noteLongaSquareDown",
-    "space"
+    "space",
+
+    //Jianpu
+    "keysig_1_Jianpu",
+    "keysig_Equal_Jianpu",
+    "keysigFlat_G_Jianpu",
+    "keysigFlat_A_Jianpu",
+    "keysigFlat_B_Jianpu",
+    "keysigFlat_C_Jianpu",
+    "keysigFlat_D_Jianpu",
+    "keysigFlat_E_Jianpu",
+    "keysig_G_Jianpu",
+    "keysig_A_Jianpu",
+    "keysig_B_Jianpu",
+    "keysig_C_Jianpu",
+    "keysig_D_Jianpu",
+    "keysig_E_Jianpu",
+    "keysig_F_Jianpu",
+    "keysigSharp_C_Jianpu",
+    "keysigSharp_F_Jianpu",
+    "noteRestJianpu",
+    "noteDoJianpu",
+    "noteReJianpu",
+    "noteMiJianpu",
+    "noteFaJianpu",
+    "noteSoJianpu",
+    "noteLaJianpu",
+    "noteSiJianpu",
+    "dotJianpu",
+    "beamJianpu",
+    "timeSig_0_Jianpu",
+    "timeSig_1_Jianpu",
+    "timeSig_2_Jianpu",
+    "timeSig_3_Jianpu",
+    "timeSig_4_Jianpu",
+    "timeSig_5_Jianpu",
+    "timeSig_6_Jianpu",
+    "timeSig_7_Jianpu",
+    "timeSig_8_Jianpu",
+    "timeSig_9_Jianpu",
+
+    //    end Jianpu
 } };
 
 constexpr const std::array<const char*, size_t(SymId::lastSym) + 1> SymNames::s_symUserNames { {
```

Comment:
See the comment in fonts/smufl/glyphnames.json; we should separate the Jianpu font logic from SMuFL font logic, because they are simply different things. 
In any case, the length of `s_symUserNames` must be in sync with that of `s_symNames`; actually, it _is_ in sync because the length is specified in `std::array<const char*, size_t(SymId::lastSym) + 1>`, but the given list does not have the correct length because the new entries were not added to it.